sklearn/metrics/_classification.py

```diff
@@ -2504,3 +2504,163 @@ def brier_score_loss(y_true, y_prob, *, sample_weight=None, pos_label=None):
             raise
     y_true = np.array(y_true == pos_label, int)
     return np.average((y_true - y_prob) ** 2, weights=sample_weight)
+
+
+def calibration_error(y_true, y_prob, sample_weight=None, norm='l2',
+                      n_bins=10, strategy='uniform', pos_label=None,
+                      reduce_bias=True):
+    """Compute calibration error of a binary classifier.
+
+    Across all items in a set of N predictions, the calibration error measures
+    the aggregated difference between (1) the average predicted probabilities
+    assigned to the positive class, and (2) the frequencies
+    of the positive class in the actual outcome.
+
+    The calibration error is only appropriate for binary categorical outcomes.
+    Which label is considered to be the positive label is controlled via the
+    parameter pos_label, which defaults to 1.
+
+    Read more in the :ref:`User Guide <calibration>`.
+
+    Parameters
+    ----------
+    y_true : array-like of shape (n_samples,)
+        True targets of a binary classification task.
+
+    y_prob : array-like of (n_samples,)
+        Probabilities of the positive class.
+
+    sample_weight : array-like of shape (n_samples,), default=None
+        Sample weights.
+
+    norm : {'l1', 'l2', 'max'}, default='l2'
+        Norm method. The l1-norm is the Expected Calibration Error (ECE),
+        and the max-norm corresponds to Maximum Calibration Error (MCE).
+
+    n_bins : int, default=10
+       The number of bins to compute error on.
+
+    strategy : {'uniform', 'quantile'}, default='uniform'
+        Strategy used to define the widths of the bins.
+
+        uniform
```

Comment:
```suggestion
        - 'uniform':
```
Fix format to meet sklearn docstrings format, like [here](https://github.com/scikit-learn/scikit-learn/blob/f3f51f9b611bf873bd5836748647221480071a87/sklearn/linear_model/_stochastic_gradient.py#L1790)